src/mrpro/data/_KData.py

```diff
@@ -162,7 +161,7 @@ def reshape_acq_data(data):
 
         try:
             shape = ktraj.broadcasted_shape
-            torch.broadcast_shapes(kdata.shape, shape)
+            torch.broadcast_shapes(kdata[:, 0].shape, shape)
```

Comment:
Why is this needed?

Reply:
Sanity check if trajectory and kdata match

Reply:
actually seems to work without as well. removed it now